lib/utils.pm

```diff
@@ -836,6 +836,11 @@ sub addon_license {
     push @tags, (get_var("BETA_$uc_addon") ? "addon-betawarning-$addon" : "addon-license-$addon");
   license: {
         do {
+            # Soft-fail on SLE-15 if license is not there.
+            if (sle_version_at_least('15') && check_var('DISTRI', 'sle') && !check_screen \@tags) {
```

Comment:
`loadtest "installation/addon_products_sle";` and `loadtest "installation/dud_addon";` are the only testsuites which are using `addon_license` 
These two modules are only executed in sle, so I think you can get rid of `check_var('DISTRI', 'sle')`

Reply:
Nice catch. Changed.